content/en/users/Rucio/Rucio-Admin/_index.md

```diff
@@ -0,0 +1,368 @@
+---
+title: "Rucio Admins"
+type: docs
+linkTitle: "Rucio Admins"
+weight: 30
+description: >-
+     Information for Rucio admins to better understand and perform actions for their VO.
+---
+
+Within Rucio there are several levels of Admins.
+There are the super admins which are the staff which run Multi-VO Rucio.
+Then there are VO specific admins which will look after the day to day of thier VO.
+Below are some of the tasks which VO admins will need to do to set up and maintain their VO.
+
+## Creating Accounts, Identities, and Quotas
+
+To add new users within your VO, you will need to communicate with Rucio as the VO admin.
+Then using the rucio-admin commands, create a new account and add identities to the account.
+The account is a username with no permissions, or authentication methods.
+The identities bind authentication methods and permissions to the account.
+The account you want to create identities for is input as an argument.
+Accounts will have different permissions and access (such as how much data they can store on a particular [RSE](https://rucio.readthedocs.io/en/latest/overview_Rucio_Storage_Element.html)).
+
+### CLI Example
+
+```shell
+$ rucio-admin account add --type USER --email jdoe@email.com jdoe
+
+Added new account: jdoe
+
+$ rucio-admin identity add --account jdoe --type USER --id userjdoe --email jdoe@email.com --password jdoepass
+
+Added new identity to account: userjdoe-jdoe
+
+$ rucio-admin account set-limits jdoe storagesite1 100GB
+
+Set account limit for account jdoe on RSE storagesite1: 100.000 GB
+```
+
+### Python Client Example
+
+```python
+$ python
+>>> from rucio.client import Client
+>>> CLIENT = Client()
+>>> CLIENT.add_account('jdoe', 'USER', 'jdoe@email.com')
+True
+>>> CLIENT.add_identity('jdoe', 'USER', 'jdoe@email.com')
+True
+>>> CLIENT.set_account_limit('jdoe', 'storagesite1', 107374182400, 'global')
+True
+```
+
+## Creating RSEs
+
+[Rucio Storage Elements](https://rucio.readthedocs.io/en/latest/overview_Rucio_Storage_Element.html)
+(RSEs) are how Rucio represents the physical storage available to your VO. As
+with many aspects of Rucio there are a lot of optional attributes that can be
+set for an RSE, but as a minimum a protocol for transfers need to be added
+before it can be used.
+
+### CLI Example
+
+```shell
+$ rucio-admin rse add NEW_RSE
+
+Added new deterministic RSE: NEW_RSE
+
+$ rucio-admin rse add-protocol --hostname test.org --scheme gsiftp --prefix '/filepath/rucio/' --port 8443 NEW_RSE --domain-json '{"wan": {"read": 1, "write": 1, "third_party_copy": 0, "delete": 1}, "lan": {"read": 1, "write": 1,"third_party_copy": 0, "delete": 1}}'
+```
```

Comment:
Please split this command on multiple lines for clarity and aiming at staying under 80 chars (fine to keep the json longer if there is no other choice).

Reply:
split across more lines